src/ChunkReader.php

```diff
@@ -94,6 +96,14 @@ public function read(WithChunkReading $import, Reader $reader, TemporaryFile $te
 
         $jobs->push($afterImportJob);
 
+        // Check if the import class is batchable
+        if ($import instanceof ShouldBatch) {
+            return Bus::batch([
+                $jobs->toArray(),
```

Comment:
By doing this, it will execute as a chain with in the batch right? It won't run parallel?

Reply:
That's right. This creates a batch and returns a PendingBatch.